### PR TITLE
aws - connect resource - add set-attribute action

### PR DESCRIPTION
**Pull request by timmygrable**
Added action `set-attribute` for the connect resource. Related to https://github.com/cloud-custodian/cloud-custodian/issues/8004

### CODE REVIEW

File: c7n/resources/connect.py [hunk 1]
```diff
@@ -58,3 +59,53 @@ def process(self, resources, event=None):
...
+        permissions = ("connect:UpdateInstanceAttribute",)
+
+        def process(self, resources):
+            client = local_session(self.manager.session_factory).client('connect')
+
+            for r in resources:
+                client.update_instance_attribute(InstanceId=r["Id"],
+                    AttributeType=self.data.get("attribute_type"), Value=self.data.get("value"))
+
+            return
```
Comment (PratMis) [inline c7n/resources/connect.py:111]: I don't think this return statement is needed since we want to bubble up the exception (if any) to the user.

Comment (thisisshi) [reply c7n/resources/connect.py:111]: exceptions will bubble up anyway as there's no try/except here, the return in general is not needed for actions

**Comment by PratMis** [comment]
Looks like the workflow/CI needs an approval to run again @thisisshi.

**Review by ajkerrigan — APPROVED** [review]
Looks good to me, thanks! :+1:

**Pull request merged**